custom/icds_reports/management/commands/sql_scripts/urgent_pmo_datapull.sql

```diff
@@ -77,4 +77,9 @@ select sum(rations_21_plus_distributed) as mother_thr from agg_ccs_record where
                      ->  Parallel Seq Scan on icds_db_aww_incentive_c9fc6833563fe8049903a475  (cost=0.00..543.60 rows=6928 width=2)
                            Filter: (month = '2019-03-01'::date)
                      ->  Parallel Seq Scan on icds_db_aww_incentive_44c9a47a653b5bb720a56dcd  (cost=0.00..348.10 rows=4488 width=2)
- */
\ No newline at end of file
+ */
+
+select sum(valid_in_month) from "agg_child_health" where month='2018-03-01' and age_tranche::INTEGER between 6 and 36 and aggregation_level=1 and state_id in ('f98e91aa003accb7b849a0f18ebd7039','f9b47ea2ee2d8a02acddeeb491d3e175','a2fcb186e9be8464e167bb1c56ce8fd9','f1cd643f0df908421abd915298ba57bc', '2af81d10b2ca4229a54bab97a5150538','d982a6fb4cca0824fbde59db18d3800f','9cd4fd88d9f047088a377b7e7d144830','9051ef4f91d54a89b89c533780857e05');;
+select sum(valid_in_month) from "agg_child_health" where month='2018-03-01' and age_tranche::INTEGER between 37 and 72 and aggregation_level=1 and state_id in ('f98e91aa003accb7b849a0f18ebd7039','f9b47ea2ee2d8a02acddeeb491d3e175','a2fcb186e9be8464e167bb1c56ce8fd9','f1cd643f0df908421abd915298ba57bc', '2af81d10b2ca4229a54bab97a5150538','d982a6fb4cca0824fbde59db18d3800f','9cd4fd88d9f047088a377b7e7d144830','9051ef4f91d54a89b89c533780857e05');;
+select sum(valid_in_month) from "agg_child_health" where month='2018-03-01' and age_tranche::INTEGER between 0 and 72 and aggregation_level=1 and state_id in ('f98e91aa003accb7b849a0f18ebd7039','f9b47ea2ee2d8a02acddeeb491d3e175','a2fcb186e9be8464e167bb1c56ce8fd9','f1cd643f0df908421abd915298ba57bc', '2af81d10b2ca4229a54bab97a5150538','d982a6fb4cca0824fbde59db18d3800f','9cd4fd88d9f047088a377b7e7d144830','9051ef4f91d54a89b89c533780857e05');;
```

Comment:
Does this intentionally do 6-36, 36-72 and 0-72, but not 0-6?

Reply:
yeah, they want it separately.

Reply:
6-36 is required to get the denominator for children eligible for THR, 37-72 is for denominator for PSE and HCM